src/Altinn.Platform/Altinn.Platform.Storage/UnitTest/TestingControllers/InstancesControllerTests.cs

```diff
@@ -351,15 +351,41 @@ public async void GetMany_OrgRequestsAllAppInstances_Ok()
             Assert.Equal(expectedNoInstances, queryResponse.TotalHits);
         }
 
-        /// <summary>
-        /// Test case: Org user requests to get all instances linked to their org.
-        /// Expected: List of instances is returned.
-        /// </summary>
+            /// <summary>
+            /// Test case: Org user requests to get multiple instances from one of their apps.
+            /// Expected: List of instances is returned.
+            /// </summary>
         [Fact]
-        public async void GetMany_OrgRequestsAllInstances_Ok()
-        {
-            // Arrange
-            string requestUri = $"{BasePath}?org=ttd";
+            public async void GetMany_OrgRequestsAllAppInstancesAlternativeScope_Ok()
+            {
```

Comment:
Looks like there is something wrong with indentation

Reply:
fixed